blocks/models/huggingface/base.py

```diff
@@ -92,30 +93,40 @@ def fit(self, dataset: List[str], labels: Optional[pd.Series]) -> None:
             self.trainer_callbacks if hasattr(self, "trainer_callbacks") else None,
         )
 
-        model, tokenizer = safe_load(
```

Comment:
I removed the part which loaded the model again from disk, but we can add it back if we want to.